chapters/data-analysis.tex

```diff
@@ -1,292 +1,488 @@
+It is completely acceptable to have folders for each task, and compartmentalize each analysis as much as needed.
+It is always better to have more code files open than to keep scrolling inside a given file.
 
-\section{Visualizing data}
-
-\textbf{Data visualization}\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data_visualization}} is increasingly popular,
-\index{data visualization}
-but a great deal of it is very low quality.\cite{healy2018data,wilke2019fundamentals}
-The default graphics settings in Stata, for example,
-are pretty bad.\sidenote{Gray Kimbrough's
-\textit{Uncluttered Stata Graphs} code is an excellent
-default replacement that is easy to install.
-\url{https://graykimbrough.github.io/uncluttered-stata-graphs/}}
-Thankfully, graphics tools like Stata are highly customizable.
-There is a fair amount of learning curve associated with
-extremely-fine-grained adjustment,
-but it is well worth reviewing the graphics manual
-to understand how to apply basic colors, alignments, and labelling tools\sidenote{
-\url{https://www.stata.com/manuals/g.pdf}}
-The manual is worth skimming in full, as it provides
-many visual examples and corresponding code examples
-that detail how to produce them.
-
-Graphics are hard to get right because you, as the analyst,
-will already have a pretty good idea of what you are trying to convey.
-Someone seeing the illustration for the first time,
-by contrast, will have no idea what they are looking at.
-Therefore, a visual image has to be compelling in the sense
-that it shows that there is something interesting going on
-and compels the reader to figure out exactly what it is.
-A variety of resources can help you
-figure out how best to structure a given visual.
-The \textbf{Stata Visual Library}\sidenote{
-\url{https://worldbank.github.io/Stata-IE-Visual-Library/}}
- has many examples of figures that communicate effectively.
+To accomplish this, you will need to make sure that you have an effective data management system, including naming, file organization, and version control.
+Just like you did with each of the analysis datasets, name each of the individual analysis files descriptively.
+Code files such as \path{spatial-diff-in-diff.do}, \path{matching-villages.do}, and \path{summary-statistics.do} 
+are clear indicators of what each file is doing, and allow you to find code quickly.
+If you intend to numerically order the code as they appear in a paper or report, 
+leave this to near publication time.
+
+% Self-promotion ------------------------------------------------
+Our team has created a few products to automate common outputs and save you 
+precious research time.
+The \texttt{ietoolkit} package includes two commands to export nicely formatted tables.
+\texttt{iebaltab}\sidenote{\url{https://dimewiki.worldbank.org/wiki/Iebaltab}} creates and exports balance tables to excel and {\TeX}. 
```

Comment:
We have consistently used LaTeX before. Why only TeX here?

Reply:
Changed to LaTeX in c80a095. However, we should look up the difference between the two and be more precise in their usage.

Reply:
Ok, created issue in https://github.com/worldbank/d4di/issues/323. @luizaandrade , feel free to resolve this thread once you've seen this comment.